docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -57,7 +57,13 @@ Here is a summary of the naming convention followed in OSCAR:
   also being consistent.
   For compatibility with standard Julia, while staying consistent internally,
   we also provide aliases (using `AbstractAlgebra.@alias`) for various standard
-  Julia functions, e.g. `is_one` as alias for `isone`
+  Julia functions, e.g. `is_one` as alias for `isone`.
+- A function returning the number of some things should be named `number_of_things`,
+  alternatively it can be named `n_things` with an alias to `number_of_things`.
```

Comment:
This reads to me as if everything should have a `number_of_things` function and may additionally have a `n_things` alias. Is this what you intended to say here? I understood it before that every `number_of_things` function should have an `n_things` alias.

Reply:
Ah, I didn't notice you swapped the alias direction of only some functions in #3364. I would find it way more consistent to have all aliases the same there, i.e. `number_of_` is the default, and then `n_` is the alias. But that's a different story. The formulation here now matches the reality of the implementation

Reply:
I think this paragraph is fine now -- at least it seems to match what @benlorenz described and I am fine with it, too.